test/testcafe/spec/SessionStorage_spec.js

```diff
@@ -4,7 +4,9 @@ import xhrSessionExpried from '../../../playground/mocks/data/idp/idx/error-sess
 import xhrIdentify from '../../../playground/mocks/data/idp/idx/identify';
 import xhrSuccess from '../../../playground/mocks/data/idp/idx/success';
 import xhrMagicLinkExpired from '../../../playground/mocks/data/idp/idx/terminal-return-expired-email';
+import xhrIdentifyWithNoAppleCredentialSSOExtension from '../../../playground/mocks/data/idp/idx/identify-with-no-sso-extension';
```

Comment:
should we also update this test `DeviceSSOExtensionView_spec.js` where it can check for stateToken as well ? 
`'cancels transaction when the authenticator does not exist'`

Reply:
That is already being tested with the new test in this file: https://github.com/okta/okta-signin-widget/pull/2047/files#diff-d5619ca161ed5c74752d62d68598cc9038e1a588eee3755c9f3a103ed3ad43e0R223

I see this file as testing the stored stateToken, and it may pull in various flows to do so. Then the individual view test classes can just focus on testing their UI.

Reply:
Issue is other developers are not aware of this at all.. so it's easy to miss and they don't care about this file `SessionStorage`..

It's okay as of now. We should come up with a mechanism that every flow first test `stateToken` before and after the execution.. something like `checkConsoleMessages`

Reply:
Right, I added a check to the device tests https://github.com/okta/okta-signin-widget/pull/2047/commits/ab5d7316cd2cc06ffc998661371ccc491ef4daf6